terraform/environments/core-network-services/firewall-rules/production_rules.json

```diff
@@ -194,5 +194,12 @@
     "destination_ip": "0.0.0.0/0",
     "destination_port": "5721",
     "protocol": "UDP"
+  },
+  "parole_board_to_ppud_production_https": {
```

Comment:
Sorry nitpicking on naming, could this be `parole_board_to_mp_hmpps_production_https`